src/app/modules/ix-forms/components/ix-combobox/ix-combobox.component.ts

```diff
@@ -128,6 +129,13 @@ export class IxComboboxComponent implements ControlValueAccessor, OnInit {
           if (this.selectedOption) {
             this.filterChanged$.next('');
           }
+        } else if (this.initialValue?.value) {
+          /**
+           * Workaround!
+           * TODO: Redesign ix-combobox provider to reliably fetch initial values without needing fake options
```

Comment:
This part is not necessary, we fixed in https://ixsystems.atlassian.net/browse/NAS-123407